css/search.css

```diff
@@ -1,5 +1,7 @@
 .jobs-unified-top-card__job-insight--highlight,
-div.jobs-unified-top-card__primary-description > div > span:last-child,
+div.jobs-unified-top-card__primary-description-without-tagline
```

Comment:
Selecting the last child span element of a parent div element of a parent div with this new classname.